social.md

```diff
@@ -11,134 +11,273 @@ support:
+
+All of these details are hidden from the framework integration. The framework integration can implement an ajax-based workflow or a page-based workflow as it sees fit.
+
+There are two primary responsibilties of the framework integration:
+
+1. Obtain a URL to interact with the Client API
+2. Validate and parse a Stormpath JWT assertion response
+
+## Obtain a URL to interact with the Client API
+
+The Social Provider login flow starts with a call to the `/authorize` endpoint of the Client API. There are two ways to obtain the proper URL:
+
+1. Get the URL directly from the Client API's `/login` model
+    * recommended as the easiest way to get the fully qualified URL to start the social login flow
+2. Construct the URL and add it to the framework integration's `/login` model.
+    * useful if the framework integration has extended the `/login` model beyond the framework specification or uses any custom form fields
+
+### Obtain the `/authorize` URL via the Client API `/login` model
+
+In order to interact with the Client API, you need the `webConfig` from the backend `/application` endpoint:
+
+`GET /applications/:applicationId?expand=webConfig`
+
+This returns a response like:
+
+```
+{
+    ...
+    "webConfig": {
+        ...
+        "dnsLabel": "elegant-lynx",
+        "domainName": "elegant-lynx.apps.dev.stormpath.io",
+        ...
+        "login": {
+            "enabled": true
+        },
+        ...
+        "status": "ENABLED",
+        ...
+    }
+    ...
+}
+```
+
+Once you have the `webConfig.domainName`, you can get the `/login` model from the Client API: `https://elegant-lynx.apps.dev.stormpath.io/login`.
+
+The response from the Client API will look like this:
+
+```
+{
+   "form":{
+      ...
+   },
+   "accountStores":[
+      {
+         "authorizeUri":"https://<webConfig domain name>/authorize?response_type=stormpath_token&account_store_href=<url encoded linkedin dir href>",
+         "name":"LinkedIn",
+         "provider":{
+            "href":"<linkedin dir href>/provider",
+            "providerId":"linkedin",
+            "clientId":"<client id>",
+            "scope":"r_basicprofile r_emailaddress"
+         },
+         "href":"<linkedin dir href>"
+      },
+      {
+         "authorizeUri":"https://<webConfig domain name>/authorize?response_type=stormpath_token&account_store_href=<url encoded facebook fir href>",
+         "name":"Facebook Dir",
+         "provider":{
+            "href":"<facebook dir href>/provider",
+            "providerId":"facebook",
+            "clientId":"<client id>",
+            "scope":"public_profile email"
+         },
+         "href":"<facebook dir href>"
+      }
+   ]
+}
+```
+
+The available `/authorize` URL(s) can be used as-is to kick of the social login flow. There are additional parameters that can be added to the `/authorize` URL as described [here](#optional-query-parameters).
+
+### Construct the `/authorize` URL
```

Comment:
Let's say you've customized the login model in your integration. Maybe you have the placeholder text for login set to a different value than the default. This would not automatically be reflected by the client API. Therefore, I thought it was important to specify that the developer could construct the `/authorize` url and still use their own login model.